### PR TITLE
Arch Linux compatibility

### PR DESCRIPTION
**Pull request by priezz**

### CODE REVIEW

**Comment by Tomohiro** [comment]
@priezz Nice, thanks!

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Command-line AirPlay video client for Apple TV
 Requirements
 -------------------------------------------------------------------------------
 
-- OS X or Ubuntu
+- OS X, Ubuntu, Arch Linux ([nss-mdns](https://www.archlinux.org/packages/extra/x86_64/nss-mdns/) package is required)
```
Comment (Tomohiro) [inline README.md]: Could you please write detailed command to install like [Ubuntu](https://github.com/Tomohiro/airplayer#ubuntu)?

Comment (priezz) [inline README.md]: Sure, it's done.

**Comment by Tomohiro** [comment]
Thanks!

**Pull request merged**